features/steps/assertor.go

```diff
@@ -0,0 +1,122 @@
+package steps
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+)
+
+// ElasticSearchCall represents a call to ElasticSearch, containing
```

Comment:
fyi Elasticsearch is one word - the database is called Elasticsearch and it is Elastic companies search engine

Reply:
thanks, I will rename it